src/components/signupform/SignupForm.js

```diff
@@ -11,28 +11,32 @@ function Form() {
     password: "",
   });
 
-  const { addUser, signup, firebaseError } = useAuthSignUp();
+  const {  signup, firebaseError } = useAuthSignUp();
   const [error, setError] = useState({});
   const router = useRouter();
 
   const handleChange = (e) => {
     setUser({ ...user, [e.target.name]: e.target.value });
   };
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
 
     const errors = regexValidation(user);
-
     if (Object.keys(errors).length === 0) {
       signup(user);
+<<<<<<< Updated upstream
```

Comment:
Seems like you forgot to do the merge changes.

Reply:
@jjleigh done